File: code/Kotlin/libs/annotation_processing/apt/compiler/src/main/java/inject/classname/OtherClassName.java
package inject.classname;

import com.squareup.javapoet.ClassName;

/**
 * @author yuansui
 * @since 2019/11/18
 */
public interface OtherClassName {
    ClassName TYPE_TOKEN = ClassName.get("com.google.gson.reflect", "TypeToken");
    ClassName TYPE = ClassName.get("java.lang.reflect", "Type");
    ClassName EXCEPTION = ClassName.get("java.lang", "Exception");
}

File: code/Kotlin/libs/annotation_processing/kapt/compiler/src/main/java/inject/compiler/classname/OtherClassName.kt
package inject.compiler.classname

import com.squareup.kotlinpoet.ClassName

/**
 * @author yuansui
 * @since 2019/11/18
 */
object OtherClassName {
    val TYPE_TOKEN = ClassName("com.google.gson.reflect", "TypeToken")
    val TYPE = ClassName("java.lang.reflect", "Type")
    val EXCEPTION = ClassName("java.lang", "Exception")
}
